dlc-manager/src/subchannel/mod.rs

```diff
@@ -395,9 +408,9 @@ pub struct ClosingSubChannel {
 }
 
 /// Provides the ability to access and update Lightning Network channels.
-pub trait LNChannelManager<SP>: ChannelMessageHandler
+pub trait LNChannelManager<K: Deref>: ChannelMessageHandler
```

Comment:
nit: feels like `SP` was a pretty good generic id for `SignerProvider`?

Reply:
Good catch! I think this is a remnant of a previous version that involved more type parameters.